src/support/command/base-command.js

```diff
@@ -71,7 +71,7 @@ class BaseCommand extends Command {
   
   executeHttp({ execute, onResolve, onReject = handleErrors, options: { resolveStatus = [] } }) {
 
-    if (!isURLValid()) {
+    if (!config.isURLValid()) {
```

Comment:
This was done to mock the `isUrlValid` method. Not sure if there's another way around it.